src/client/src/components/pages/participants/viewParticipants/index.jsx

```diff
@@ -27,6 +22,32 @@ export default class ViewParticpants extends Component {
   clear = () => {
     this.setState({ search: "" });
   };
+  
+// axios to make requests from backend.. 
+  componentDidMount = async () => {
+    try {
+      const data = await axios("/participants");
+      const finalData = data.data.getParticipants;
+      const array = [["Full Name", "Date Of Birth", "Email", "Action"]];
+      finalData.map(row =>
+        array.push([
+          row.fullname,
+          row.date_of_birth.split('T')[0],
+          row.email,
+          <Fragment>
+            <i className="fas fa-trash-alt" />
+            <Link to="/participant/details">
+              <i className="fas fa-info-circle" />
+            </Link>
+          </Fragment>
+        ])
+      );
+      this.setState({ rows: array });
+    } catch (err) {
+      console.log(err); // waiting for boundery error handling
+    }
+  };
+
   render() {
     return (
       <React.Fragment>
```

Comment:
You have already imported `Fragment` from `React`, no need for `<React.Fragment>`, change it to `<Fragment>` or `<>`.

Reply:
I do it :wink: